src/components/BMDashboard/shared/CheckTypes.jsx

```diff
@@ -30,8 +30,14 @@ function CheckTypes({ type }) {
   };
 
   // Filter the data based on the search query
+  const normalizeString = str =>
+    str
+      .trim()
+      .replace(/\s+/g, ' ')
```

Comment:
To make the search work for queries like "helloworld" matching "Hello World" (with or without spaces), update the normalizeString function to remove all spaces:

```suggestion
      .replace(/\s+/g, '')
```
With this change, searching for "helloworld" will match "Hello World", "hello world", or "  hello    world   ".